pulp_smash/tests/rpm/api_v2/test_unassociate.py

```diff
@@ -0,0 +1,114 @@
+# coding=utf-8
+"""Test unassociation of RPM unit types.
+
+This module assumes that the tests in
+:mod:`pulp_smash.tests.platform.api_v2.test_repository` and
+`pulp_smash.tests.platform.api_v2.test_sync_publish` hold true.
+
+http://pulp.readthedocs.org/en/latest/dev-guide/integration/rest-api/content/associate.html#unassociating-content-units-from-a-repository
+"""
+
+from __future__ import unicode_literals
+
+try:  # try Python 3 import first
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin  # pylint:disable=C0411,E0401
+
+from pulp_smash import api, utils
+from pulp_smash.constants import REPOSITORY_PATH, RPM_FEED_URL
+from pulp_smash.tests.rpm.api_v2.utils import gen_repo
+
+
+def _remove_unit(client, repo_href, type_id, unit_id):
+    """Remove a unit from a repository.
+
+    :param pulp_smash.api.Client client: object to make calls to API
+    :param repo_href: url of the repo the unit is associated to
+    :param type_id: type of unit that will be removed
+    :param unit_id: id of the unit to be removed
+    :return: response from server
+    """
+    rm_path = urljoin(repo_href, 'actions/unassociate/')
+    if type_id is 'rpm':
```

Comment:
`is` evaluates object identity. Roughly, it checks to see whether two variables point to the same location in memory. I think a comparison of values with `==` may be what is intended here.